README.md

```diff
@@ -30,8 +30,9 @@ Interviews
  * A *Linked List* is a linear collection of data elements, called nodes, each
    pointing to the next node by means of a pointer. It is a data structure
    consisting of a group of nodes which together represent a sequence.
- * **Singly-linked list**: linked list in which nodes have pointers to the next node
- * **Doubly-linked list**: linked list in which nodes have pointers to *both* the previous node and the next node
+ * **Singly-linked list**: linked list in which nodes have pointers to the next node and the last node point to null
+ * **Doubly-linked list**: linked list in which nodes have two pointers which points to *both* the previous node and the next node
```

Comment:
"linked list in which each node has two pointers *p*, *n* such that *p* points to the previous node and *n* points to the next node; the last node's *n* pointer points to null"